lib/shared/string-utils.js

```diff
@@ -47,11 +41,24 @@ function getGraphemeCount(value) {
         return value.length;
     }
 
-    if (!splitter) {
-        splitter = new GraphemeSplitter();
+    if (!iterateGraphemes) {
+        if (typeof Intl !== "undefined" && typeof Intl.Segmenter !== "undefined") {
+            iterateGraphemes = Intl.Segmenter.prototype.segment.bind(new Intl.Segmenter());
+        } else {
+            const Graphemer = require("graphemer").default;
+
+            iterateGraphemes = Graphemer.prototype.iterateGraphemes.bind(new Graphemer());
+        }
```

Comment:
Do these two methods always give the same result? If we can't guarantee that, it would be better to switch to just `graphemer` now and then just `Intl.Segmenter` in the next major version (when we drop Node < 16), because we wouldn't want linting to produce different results in different environments.